navigation/ca_move_base/launch/navigate.launch

```diff
@@ -1,6 +1,6 @@
 <launch>
   <arg name="robot_name"/>
-  <arg name="slam" default="$(optenv SLAM false)"/>
+  <arg name="navigation" default="$(optenv NAVIGATION amcl)"/>
```

Comment:
amcl is a localization algorithm, not a navigation. I would recommend to change the name of the variable.

Reply:
I know but I don't have an idea now. Do you?

Reply:
maybe: "localization_package=", "use_localization=" ?